db-mojo-scorer/README.md

```diff
@@ -0,0 +1,258 @@
+# Database Scorer
+Some customers need to select rows from a Database, score and write back the predictions.Using DAI and then this program enables them to quickly build and use models.
+
+Supports following databases
+1. Postgres DB
+2. Microsoft SQL Server 12
+3. TerraData
+4. RedShift
+5. Oracle
+6. DB2
+7. Azure Data Warehouse
+8. Snowflake
+9. ElasticSearch
+ 
+## Building
+The code of the local SpringBoot scorer is a gradle project build as usual by `./gradlew build`.
+
+The resulting executable jar is located in the `build/libs` folder.
+ 
+## Running 
+- You must have a DAI license to score, so add the license (via command line) in one of the standard ways, as a parameter, as a environment vailable for example:
```

Comment:
@NidhiMehta I was wondering if this section could go to the index.md page cause Im sure all deployments require a license file